Packs/Okta/Integrations/OktaLog/OktaLog.yml

```diff
@@ -41,15 +24,24 @@ configuration:
   type: 0
   defaultvalue: 1000
   required: true
-- display: XSIAM update limit per request
-  name: events_to_add_per_request
-  type: 0
-  defaultvalue: 1000
-  required: false
 - display: First fetch from api time
```

Comment:
It should be type `short text`. see the `fetch_time` parameter in Gmail, for example. https://github.com/demisto/content/blob/master/Packs/Gmail/Integrations/Gmail/Gmail.yml#L43